### PR TITLE
Get github pruner for elevate-cpanel working

### PR DESCRIPTION
**Pull request by davelcpanelnet**
Case RE-924: Get github pruner for elevate-cpanel working

Changelog:

By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

File: .github/workflows/openstack/prune_openstack [hunk 1]
```diff
@@ -43,7 +47,7 @@ remove_stale_keys () {
     do
         if [[ $CREATED_DATE < $HAMMER_TIME ]];
         then
-            echo "deleting: $ID,$CREATED_DATE,$NAME"
+            echo "deleting: $ID,$CREATED_DATE,$NAME"s
```
Comment (cPholloway) [inline .github/workflows/openstack/prune_openstack:50]: Stray `s` at the end?

File: .github/workflows/openstack/volume_ids.txt
Comment (cPholloway) [inline .github/workflows/openstack/volume_ids.txt:1]: What is the purpose of adding this touch file? Should we just add the files that prune_openstack creates to .gitignore?

**Review by cPholloway — CHANGES_REQUESTED** [review]

File: .github/workflows/openstack/prune_openstack (hunk 1 above)
Comment (davelcpanelnet) [reply .github/workflows/openstack/prune_openstack:50]: Good catch!

File: .github/workflows/openstack/volume_ids.txt
Comment (davelcpanelnet) [reply .github/workflows/openstack/volume_ids.txt:1]: should be yanked.  Getting that out of there.

**Review by toddr — APPROVED** [review]

**Pull request merged**